lib/Metadata.php

```diff
@@ -44,6 +44,11 @@ class Metadata
     private $fieldMappings = [];
 
     /**
+     * @var bool
+     */
+    private $removeLive = true;
```

Comment:
I am somehow missing the connection that this is done automatically when the draft is deleted. Maybe `syncRemoveAcrossWorkspaces`? Although this is quite a long one... Or `syncRemoveLive`?